### ISSUE TITLE
Synchronize naming between text tree and graphical output of EXPLAIN ANALYZE in v21.1

### ISSUE THREAD

**Issue by ianjevans** [cockroachdb/cockroach#62912]
**Is your feature request related to a problem? Please describe.**
The name of the fields and properties in the output of `EXPLAIN ANALYZE (PLAN)` and `EXPLAIN ANALYZE (DISTSQL)` doesn't match in v21.1.

For example, in the text tree output of `PLAN` the names are:
```
KV rows read: 125,000
```

While in the graphical `DISTSQL` viewer it's:
```
KV tuples read
```

"KV rows" is easier to understand.

**Describe the solution you'd like**
The field and property names should match between the text tree and the `DISTSQL` diagram.

**Additional context**

For this query in MovR:
```
SELECT city, AVG(revenue) FROM rides GROUP BY city;
```

This is the output from `PLAN`:
```
                                        info
-------------------------------------------------------------------------------------
  planning time: 478µs
  execution time: 94ms
  distribution: full
  vectorized: true
  rows read from KV: 125,000 (21 MiB)
  cumulative time spent in KV: 180ms
  maximum memory usage: 1.1 MiB
  network usage: 2.9 KiB (25 messages)

  • group
  │ cluster nodes: n1, n2, n3
  │ actual row count: 9
  │ estimated row count: 9
  │ group by: city
  │ ordered: +city
  │
  └── • scan
        cluster nodes: n1, n2, n3
        actual row count: 125,000
        KV rows read: 125,000
        KV bytes read: 21 MiB
        estimated row count: 125,000 (100% of the table; stats collected 1 day ago)
        table: rides@primary
        spans: FULL SCAN
(24 rows)

Time: 95ms total (execution 95ms / network 0ms)
```

And the [graphical output from `DISTSQL`](https://cockroachdb.github.io/distsqlplan/decode.html#eJzcmE1u4zYUx_c9BcFVBlViUV-WtfJkkLbBJPYgHwUGVRAw0qsjjCy6JJ1xGuQmBXqKbrrsIXqAnqSgZE9sWaJlR0mL2QiSHh_13o_8_83kAYtfUhzg86OTo3cXKErkvYHo3WiPwx1kU3iDvjsbniKexCDQ92fDyw_o8GM-DBs4YzEM6BgEDn7CBBvYwga28ZWBJ5xFIATjKvSQDzyOZzgwDZxkk6lUr68MHDEOOHjAMpEp4ABf0JsUzoDGwDsmNnAMkiZpPn1eQX_CkzHl6tvnE5qJAO13QkzHQgKP6TjEnRCH4SzqhuGM9A7CcEZddf251w_DmWmG4cw3F3flC3lQqWaIEc1iZCEmb4ELbODhVAaobxl9omp6_yOSyRgC5LhjUTxHLJOQyYRl85D51x_zkJxOUhCIA40D5BDDc50icHMvv7z3DnroNDnEBr6hMroFgdhUTtRXHUV1PsfTu3yWq0cDF6_mMIWkI8ABeTSaA387GnEYUcl4x17l3VdfHvIYOMQByp_eDj5eD4YX14PLk5O9PnmjVuHydK9vqbt3w8vBxfweZhBNl3DYOahybxWtuaWungq9uUe3VNyu1Xj1-NS5Vdv50zys6Kg8z7fFRBo83vPwnF-eXh8rQLZ6OoMsBq7yDNS3On27AprrFpuojM1ax2bXbga7AZJpVgWlkseA7bNJp7eKoly4U123u153r7ZuZ6Vu0tw1rEau0QnxDROSZblj-P_8_lsYzsAPw1ncO9zsFGZahLFyn5QJRLMRpCAW_kOpuigTiuHvP9XY6IcG_nM0nzX3H1L4j8Z-ervYj-eu20_3gGxpP15ZqHX2s2HllvTlfmX2Q9q1H_-17cfqVsu4AhtpaD9WcxmTpjJu-7d_vy1raHiIcMkuKvZJGyr26xeObLNwS9vU-R-o2GtRxVa7Ku6-toq9XuNDRL2l1x8iKro9AzFhmYDSYaJ6ZlPxgngEBV_BpjyCD5xF-WeKx2Gel7-IQcgiSoqH46wIqQKXk0k5mSwnWyvJJK9GVh9l_HzTjekMjWHM-D2iacoiKtV62SZ6n-tNhUXEFWIUJ-LT8iATVSmyYtdaCv82XfRqu8hAfmb8E0qphCy6D5DbnZv5IvKZJnL1GBGDAJ7QNPmVLvVPSnlfjCiC5E61Zy0FF2a0iNmulTe_iI9BCDpaHfJCbLzmbBzf35GNt4EN0bEhbbEh62wsrXTsVTblZFuvO1MvPEeb7eqT3R1Vu77fLVK_pvPlrl7UZ-13z3y5_a5n05YXdHVsiL0Bjq2F47cEx94WTr0ZlO1-fuKqtntrd7s317sw17vwtpHtrtvfdTWWZm9aYZ2lWd0XtLSulo2v9xX_NXxFx3WjcrS24jgvZyt6NPW2svbPqlc7J1XIX99ES2cB7fqazzkKWG39bFTopvffe4qzgY2rYeO53iY2TvO_bq4ev_k3AAD__yqgOLs=).

**Comment by ianjevans** [comment]
Tuples -> rows fix in #62910.

**Comment by yuzefovich** [comment]
Fixed in #62910 on master and backported the fix in #62956 to 21.1 branch.
